src/main/java/com/softserve/actent/controller/UserController.java

```diff
@@ -53,15 +54,21 @@ public IdDto addUser(@Validated @RequestBody UserRegistrationDto userRegistratio
 
     @PutMapping(value = "/users/{id}")
     @ResponseStatus(HttpStatus.OK)
-    public UserDto updateUserById(@Validated @RequestBody UserSettingsDto userSettingsDto, @PathVariable @NotNull @Min(value = NumberConstants.ID_MIN_VALUE, message = StringConstants.USER_ID_SHOULD_BE_GREATER_THAN_ZERO) Long id) {
+    public UserDto updateUserById(@Validated @RequestBody UserSettingsDto userSettingsDto,
```

Comment:
Are you sure that you need @Validated here but not just @Valid ?

Reply:
I know that I can use @Valid here. But we decide to use @Validated as it have the same functionality as @Valid and plus some more.
Is it bad practice to use it when we don't use that additional functionality?